docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -15,22 +15,21 @@
     specific language governing permissions and limitations
     under the License.
 
-Extra Packages Reference
-''''''''''''''''''''''''
+Reference for package extras
+''''''''''''''''''''''''''''
 
-Here's the list of the :ref:`subpackages <installation:extra_packages>` and what they enable.
+Here's the list of all the :ref:`extra dependencies <installation:airflow_extra_dependencies>`.
 
-The entries with ``*`` in the ``Providers`` column indicate that one or more provider will be installed
-automatically when those extras are installed. In those cases, there is a dependency between corresponding
-provider packages and ``apache-airflow`` package (the provider package depends on ``apache-airflow>=2.0.0``).
+Most of the the extras result in installing additional provider packages.
+
+The entries with ``*`` in the ``Providers`` column indicate that one or more provider packages will be
+installed automatically when those extras are used.
 For ``provider`` extras - they usually install single ``provider`` package, but for extras that are groups
 of other extras (for example ``all`` or ``devel_all`` or ``all_dbs``) there might be more than one provider
 installed together with the extra.
 
 The entries with ``*`` in the ``Preinstalled`` column indicate that those extras (with providers) are always
-pre-installed when Airflow is installed. In this case dependencies are reverted - the ``apache-airflow``
-package depends on the corresponding providers packages. This is in order to avoid circular dependency that
-can be reported by some tools (even if it is harmless).
+pre-installed when Airflow is installed.
```

Comment:
Should ``Preinstalled`` column be added to the other tables?

Reply:
No neeed. Only the 4 `provider` packages are pre-installed. All others have to be explicitly added